scripts/ngless-cwl/cwl/ngless-count.cwl

```diff
@@ -6,49 +6,48 @@
 cwlVersion: "cwl:v1.0"
 
 class: CommandLineTool
-baseCommand: ['./ngless-count.py']
+baseCommand: ['ngless-count.py']
+
+requirements:
+- $import: ngl-types.yml
 
 doc: |
   None
 
 inputs:
-  
   input:
-    type: str
-  
+    type: File
     doc: SAM/BAM/CRAM file to count reads on
     inputBinding:
-      prefix: --input 
+      prefix: --input
 
   output:
-    type: str
-  
+    type: string
     doc: Output file/path for results
```

Comment:
NGLess can generate different outputs depending on the extension of the file. Having an option for this makes workflows a little bit more flexible.